sklearn/linear_model/tests/test_coordinate_descent.py

```diff
@@ -302,7 +303,30 @@ def test_lasso_cv_positive_constraint():
     assert min(clf_constrained.coef_) >= 0
 
 
-# FIXME: 'normalize' to be removed in 1.2
+def _scale_alpha(estimator, n_samples):
+    """"Rescale the parameter alpha from when the estimator is evoked with
+    normalize set to True to when it is evoked in a Pipeline with normalize set
+    to False and with a StandardScaler."""
+    if 'alpha' not in estimator.get_params():
+        return
+
+    model_name = estimator.__class__.__name__
+    if model_name in ['Lasso', 'LassoLars', 'MultiTaskLasso']:
+        alpha = estimator.alpha * np.sqrt(n_samples)
+    if model_name in ['Ridge', 'RidgeClassifier']:
+        alpha = estimator.alpha * n_samples
+    if model_name in ['ElasticNet', 'MultiTaskElasticNet']:
+        if estimator.l1_ratio == 1:
+            alpha = estimator.alpha * np.sqrt(n_samples)
+        if estimator.l1_ratio == 0:
+            alpha = estimator.alpha * n_samples
+
```

Comment:
```suggestion
        elif estimator.l1_ratio == 0:
            alpha = estimator.alpha * n_samples
        else:
             # To avoid silent errors in case of refactoring
             raise NotImplementedError  # noqa

```